lacework-agent/Chart.yaml

```diff
@@ -14,4 +14,4 @@ maintainers:
 - email: info@lacework.net
   name: lacework-support
 name: lacework-agent
-version: 6.9.0
+version: 6.7.6
```

Comment:
Are we intentionally modifying this value? My understanding is that we're publishing the artifacts but not changing the default version of the chart